proxy.c

```diff
@@ -184,13 +185,21 @@ void sigterm_handler(int signal) {
     exit(0);
 }
 
+int set_reuseaddr(int sock)
```

Comment:
In this context you are right, I customized the proxy for my use in my own branch and had to re-use it so I made it a function.

I'll just add =1 to optval for your branch.